horovod/common/common.h

```diff
@@ -45,7 +45,6 @@ using gpuStream_t = cudaStream_t;
       throw std::logic_error(std::string("GPU Error:") + cudaGetErrorString(cuda_result));  \
     }                                                                                       \
   } while (0)
-#endif
```

Comment:
great, the build with HAVE_ROCM must have been broken earlier because of changes here